source/routing/raptor_api.cpp

```diff
@@ -1190,21 +1137,22 @@ pbnavitia::Response make_isochrone(RAPTOR &raptor,
                                    georef::StreetNetwork & worker,
                                    const type::RTLevel rt_level,
                                    int max_duration, uint32_t max_transfers, bool show_codes) {
-    pbnavitia::Response response;
+
+    PbCreator pb_creator(raptor.data,pt::not_a_date_time, null_time_period, show_codes);
 
     bt::ptime datetime;
-    auto tmp_datetime = parse_datetimes(raptor, {datetime_timestamp}, response, clockwise);
-    if(response.has_error() || tmp_datetime.size() == 0 ||
-       response.response_type() == pbnavitia::DATE_OUT_OF_BOUNDS) {
-        return response;
+    auto tmp_datetime = parse_datetimes(raptor, {datetime_timestamp}, pb_creator, clockwise);
+    if(pb_creator.has_error() || tmp_datetime.size() == 0 ||
+            pb_creator.has_response_type(pbnavitia::DATE_OUT_OF_BOUNDS)) {
+        return pb_creator.get_response();
```

Comment:
you may use std::move(pb_creator.get_response())

A little example : https://ideone.com/CExZqQ